downstream/modules/platform/con-gw-pluggable-authentication.adoc

```diff
@@ -4,14 +4,14 @@
 
 = Pluggable authentication
 
-Authentication is the process of verifying a user's identity to Ansible Automation Platform, which means establishing that a user is who they say they are. This can be done in a number of ways but would traditionally be associated with a username and password.
+Authentication is the process of verifying a user's identity to the {PlatformNameShort} (that is, to establish that a user is who they say they are). This can be done in a number of ways but would traditionally be associated with a `username` and `password`.
 
-In Ansible Automation Platform 2.4, you could configure external authentication sources such as LDAP or SAML, in automation controller. Ansible Automation Platform 2.5 uses a pluggable authentication system with a configuration wizard that provides a common, simplified method of configuring different types of authenticators, such as LDAP and SAML. The pluggable system also allows you to  configure multiple authenticators of the same type.  
+In {PlatformNameShort} 2.4, you could configure external authentication sources such as LDAP or SAML, in {ControllerName}. {PlatformNameShort} {PlatformVers} uses a pluggable authentication system with a configuration wizard that provides a common, simplified method of configuring different types of authenticators, such as LDAP and SAML. The pluggable system also allows you to configure multiple authenticators of the same type.  
```

Comment:
the phrase "such as LDAP or SAML" appears twice in line 9--I would remove the first instance (including the comma) since it seems more important to give detail on the new version.

Reply:
Nice catch! Fixed.